gemhardware/devices/Makefile

```diff
@@ -55,7 +59,8 @@ LibraryDirs+=$(BUILD_HOME)/gemutils/lib/$(XDAQ_OS)/$(XDAQ_PLATFORM)
 LibraryDirs+=$(uHALROOT)/lib
 LibraryDirs+=$(XHAL_ROOT)/lib
 
-DependentLibraries =cactus_uhal_uhal xhal-client
+#FIXME hacky linking of xhal-base
+DependentLibraries =cactus_uhal_uhal xhal-client xhal-base
```

Comment:
and if not, can't you remove `cactus_uhal_uhal`

Reply:
uhal is still a dependency for the AMC13-related code

Reply:
indeed, but there should be *no* `amc13` code in `gemhardware/devices` (only in `gemhardware/managers`, no?)

Reply:
indeed, you're right. Done.